Services/Common/OJS.Services.Common/ServiceConstants.cs

```diff
@@ -95,5 +95,18 @@ public static class ExecutionStrategyNames
                 public const string PlainText = "plaintext";
             }
         }
+
+        public static class BackgroundJobs
+        {
+            public static class JobNames
+            {
+                public const string EnqueuePendingSubmissionsJobName = "EnqueuePendingSubmissionsForProcessing";
+                public const string EnqueuePendingSubmissionsJobCron = "*/3 * * * *";
+                public const string EnqueuePendingSubmissionsJobAddedMessage = $"{EnqueuePendingSubmissionsJobName} job added successfully! It will run at {EnqueuePendingSubmissionsJobCron}";
```

Comment:
These are 3 different types of information. Maybe split in different classes or just put in class BackgroundJobs, but not all in JobNames

Reply:
Moved them in separate classes